assets/js/utils/energyOptions.js

```diff
@@ -8,25 +10,25 @@ export function optionStep(maxEnergy) {
   return 5;
 }
 
-export function fmtEnergy(energy, step, fmtKWh, zeroText) {
+export function fmtEnergy(energy, step, fmtWh, zeroText) {
   if (energy === 0) {
     return zeroText;
   }
   const inKWh = step >= 0.1;
   const digits = inKWh && step < 1 ? 1 : 0;
-  return fmtKWh(energy * 1e3, inKWh, true, digits);
+  return fmtWh(energy * 1e3, inKWh ? WATT_FORMAT.KW : WATT_FORMAT.AUTO, true, digits);
```

Comment:
We should stick to `W` or `kW` in this case. No auto since it would mix different units in the dropdown.

Reply:
I would suggest `kW` because that is the unit found in most households.

Reply:
No, we deliberately introduced a W/kW switch here dependent on the connected vehicle battery capacity. For e-bikes or vehicles with small batteries (hybrids) we want finer steps and therefore more digits.

Reply:
Oh, I misunderstood your message. No problem.